dev/docker-clients/persistent.hcl

```diff
@@ -0,0 +1,16 @@
+log_level = "DEBUG"
+data_dir = "/tmp/server"
```

Comment:
mind prefixing with `nomad-` so it's `/tmp/nomad-server`? Sometimes I want to wipe all nomad state from a system and a prefix makes it far easier.